torch/onnx/_internal/onnx_proto_utils.py

```diff
@@ -12,6 +14,98 @@
 from torch.onnx._internal import _beartype, jit_utils, registration
 
 
+@_beartype.beartype
+def export_as_test_case(
+    model_bytes: bytes, inputs_data, outputs_data, name: str, dir: str
+):
+    try:
+        import onnx
+    except ImportError:
+        raise ImportError(
+            "Export test case to ONNX format failed: Please install ONNX."
+        )
+
+    test_case_dir = os.path.join(dir, "test_" + name)
```

Comment:
Do we need to ensure uniqueness?

Reply:
Good catch, very likely. I first written as using datetime but later extended to user provided name. Let me check.

Reply:
I think it's ok, it overrides if exists. This only happens when name is provided by user.
If user doesn't provide the name, the path can be considered always unique.